exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -52,12 +52,18 @@
           <thead>
             <tr>
               <th>#</th>
-              <th>Address</th>
+              <th title="Drillbits in Cluster" style="cursor: help;">Address <span class="glyphicon glyphicon-info-sign" style="font-size:100%"/></th>
+              <th title="Heap Memory in use (as percent of Total Heap)" style="cursor: help;">Heap Memory Usage <span class="glyphicon glyphicon-info-sign" style="font-size:100%"/></th>
+              <th title="Direct Memory ACTIVELY in use (as percent of Peak Usage so far).&#013;The percentage is an estimate of the total because the Netty library directly negotiates with the OS for memory" style="cursor: help;">Direct Memory Usage <span class="glyphicon glyphicon-info-sign" style="font-size:100%"/></th>
```

Comment:
1. `Direct Memory ACTIVELY in use (as percent of Peak Usage so far)` -> maybe we can remove `so far`?
2. `The percentage is an estimate of the total because the Netty library directly negotiates with the OS for memory` -> not sure that information about netty is useful for the user...

Reply:
1. The problem is that the Peak Usage can go up over time. Adding 'so far' indicates that this isn't a final value.
2. We could drop it, but this was meant to indicate less confusion for users wondering why the MaxDirectMemory value is not reflected here. It does. however, make it rather verbose.

Reply:
Reduced it to
`Estimated Direct Memory ACTIVELY in use (as percent of Peak Usage)`